lib/WebService/MinFraud/Record/BillingAddress.pm

```diff
@@ -30,19 +31,19 @@ __END__
 
 This class contains the billing address data associated with a transaction.
 
-This record is returned by the Insights end point.
+This record is returned by the L<WebService::MinFraud::Model::Insights#billing_address> method.
 
 =head1 METHODS
 
 This class provides the following methods:
 
-=head2 distance_to_billing_address
+=head2 distance_to_ip_location
 
-Returns the distance from the shipping address to the billing address.
+Returns the distance in kilometers from the billing address to the IP location.
 
 =head2 is_in_ip_country
 
-Returns a boolean indicating whether or not the billing address is in the same
```

Comment:
I believe whether or not is accetable usage here, but if you feel just whether reads better than leave as such.